models/doctorDetailModel.js

```diff
@@ -0,0 +1,82 @@
+const mongoose = require("mongoose");
+
+const Joi = require("joi");
+
+const doctorSchema = new mongoose.Schema({
+  doctorName: {
```

Comment:
Please use string instead of String because String is wrapper class. Same for all below where String is present